core/services/job/models.go

```diff
@@ -277,6 +277,10 @@ type OCR2PluginType string
 const (
 	// Median refers to the median.Median type
 	Median OCR2PluginType = "median"
+
+	DKG OCR2PluginType = "dkg"
+
+	VRFOCR2 OCR2PluginType = "vrf"
```

Comment:
Naming nit: Should be `OCR2VRF` rather than `VRFOCR2`. The string should also be `ocr2vrf` for clarity.